.github/workflows/quicklogic.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           export PATH=$PATH:$PWD/.tmp
           export ROOT=$(pwd)
           export INSTALL_DIR=quicklogic-arch-defs
-          export CMAKE_FLAGS="-GNinja -DINSTALL_FAMILIES=qlf_k4n8,qlf_k6n10,pp3 -DCMAKE_INSTALL_PREFIX=${ROOT}/${INSTALL_DIR}"
+          export CMAKE_FLAGS="-GNinja -DINSTALL_FAMILIES=pp3 -DCMAKE_INSTALL_PREFIX=${ROOT}/${INSTALL_DIR}"
```

Comment:
Why qlf devices are being removed from the install package?

Reply:
This is only temporary. I'm investigating a CI failure I couldn't reproduce locally.